### PR TITLE
documentation

### PR DESCRIPTION
**Pull request by alsgregory**

### CODE REVIEW

**Comment by miklos1** [comment]
Normally the things you ought to check-in to the version control are the source code, rather than compiled binary blobs.

**Comment by alsgregory** [comment]
awesome thanks. no problem. so for this I should add .tex and .bib files?

**Comment by miklos1** [comment]
Yes, that would be better. Put them in a separate directory such as `doc/` or `docs/`. Add a makefile, too, so that people would know how to build the PDF from the .tex files (especially if there's anything special to be done).

**Comment by alsgregory** [comment]
Many Thanks,

I'll work on this!

File: docs/firedrake_mlmc_doc.aux [hunk 1]
```diff
@@ -0,0 +1,126 @@
+\relax
```
Comment (miklos1) [inline docs/firedrake_mlmc_doc.aux]: Please don't add generated files, only source files.

**Pull request merged**